README.md

```diff
@@ -17,6 +17,20 @@ If you would like to submit changes, please open a pull request.
 ```bash
 $ python -m pip install -r requirements.txt
```

Comment:
Using just `python` here is making the assumption that a user will see the python 3.6 requirement and know to use that version of the interpreter. Adding "`python` or `python3`" is probably misleading in that most people (at least on mac) will assume that we support python 2. So, I think that we should use either `python` or python3` but not both.